pkg/micropython/Makefile.dep

```diff
@@ -0,0 +1,2 @@
+USEMODULE += xtimer
+USEMODULE += stdin
```

Comment:
This file should contain the architecture blacklists nowadays if I understand correctly

Reply:
yup, added avr8 and the ESPs for now.

Reply:
ESPs or MSPs?